### ISSUE TITLE
Peer dependency of @angular/common@"^8.0.0 || ^9.0.0 || ^10.0.0

### ISSUE THREAD

**Issue by vicatcu** [videogular/ngx-videogular#64]
What's stopping this project from listing ^11.0.0 as a viable peer dependency? npm@^7 has gotten picky about this sort of thing.

**Comment by IxquitilisSaid** [comment]
Nothing's stopping us, it's just that v11 wasn't out back then.
Which version of ngx-videogular are you using? Because I published 4.0.0 yesterday and changed the peerDeps to this:
```
"peerDependencies": {
    "@angular/common": ">=8.0.0 <13.0.0",
    "@angular/core": ">=8.0.0 <13.0.0",
    "@angular/platform-browser-dynamic": ">=8.0.0 <13.0.0",
    "core-js": ">=2.5.4 <=3.15.1",
    "rxjs": ">=6.5.4",
    "zone.js": ">=0.10.2 <=0.11.4"
  },
```

**Comment by vicatcu** [comment]
@IxquitilisSaid ah, awesome, I guess i was using ^2.1.3, thanks. Is there a migration guide?

**Comment by IxquitilisSaid** [comment]
v2.1.3? I only released 2.1.0... Can you please check if you're using `@videogular/ngx-videogular`? Because there's a ngx-videogular package (without the @ scope).

Either way, yes we do! https://videogular.github.io/ngx-videogular/docs/migrating-from-videogular2/ 😄 
Please let me know if you run into any issues!

**Comment by IxquitilisSaid** [comment]
Closing this issue under the assumption that it is solved but please reopen it if that's not the case.
